README.md

```diff
@@ -7,6 +7,27 @@
 [Development guide](docs/GET_STARTED.md)
 
 ## Release notes
+[27/03/2020]  1.0.0 version:
+
+ - `@cennznet/api@1.0.0` (current stage) is working with [cennznet/cennznet:latest][cennznet/cennznet:latest];
+
+Changes made in this version:
+ 
+- Derived query for estimating fee is updated
+- New format to use fee exchange 
+    const feeExchange = {
```

Comment:
Could you please add "```js ```" code block around the code with intents, same rule applies in the below one?

Reply:
sure..done